File: app/src/main/java/com/gee12/mytetroid/views/fragments/settings/SettingsEditFragment.java
package com.gee12.mytetroid.views.fragments.settings;

import android.os.Bundle;

import com.gee12.mytetroid.R;

public class SettingsEditFragment extends TetroidSettingsFragment {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        super.onCreatePreferences(savedInstanceState, rootKey);
        setPreferencesFromResource(R.xml.prefs_edit, rootKey);

        getActivity().setTitle(R.string.pref_category_edit);
    }
}

File: app/src/main/java/com/gee12/mytetroid/ui/fragments/settings/SettingsEditFragment.kt
package com.gee12.mytetroid.ui.fragments.settings

import android.os.Bundle
import com.gee12.mytetroid.R

class SettingsEditFragment : TetroidSettingsFragment() {

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        super.onCreatePreferences(savedInstanceState, rootKey)
        setPreferencesFromResource(R.xml.prefs_edit, rootKey)
        requireActivity().setTitle(R.string.pref_category_edit)
    }
}